src/types.ts

```diff
@@ -244,18 +251,25 @@ export interface MedicationDispense {
     };
   }>;
   location: Reference;
-  type: CodeableConcept;
-  quantity: Quantity;
-  whenPrepared: any;
-  whenHandedOver: any;
-  dosageInstruction: Array<DosageInstruction>;
-  substitution: {
+  type?: CodeableConcept;
+  quantity?: Quantity;
+  whenPrepared?: any;
+  whenHandedOver?: any;
+  dosageInstruction?: Array<DosageInstruction>;
+  substitution?: {
     wasSubstituted: boolean;
     reason?: CodeableConcept[];
     type?: CodeableConcept;
   };
 }
 
+export enum MedicationDispenseStatus {
+  in_progress = "in-progress",
+  on_hold = "on-hold",
+  completed = "completed",
+  declined = "declined",
+}
```

Comment:
This looks different from the status tags being returned

Reply:
See my comments, above, these come from: https://www.hl7.org/fhir/R4/valueset-medicationdispense-status.html